pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -219,7 +219,7 @@ func (p *PriorityQueue) addNominatedPodIfNeeded(pod *v1.Pod) {
 	if len(nnn) > 0 {
 		for _, np := range p.nominatedPods[nnn] {
 			if np.UID == pod.UID {
-				klog.Errorf("Pod %v/%v already exists in the nominated map!", pod.Namespace, pod.Name)
+				klog.Infof("Pod %v/%v already exists in the nominated map!", pod.Namespace, pod.Name)
```

Comment:
Downgrade to `Infof` seems a bit aggressive, as it seems this case is unexpected.

Reply:
@resouer Thanks for the comment. Based on current solution, the message occurs more often and it's as design. So I'd put your comment aside for a while, until we come to the consensus on core idea of this PR.